lib/src/migrators/division.dart

```diff
@@ -456,9 +463,8 @@ class _DivisionMigrationVisitor extends MigrationVisitor {
   /// ParenthesizedExpression.
   void _patchParensIfAny(SassNode node) {
     if (node is! ParenthesizedExpression) return;
-    var expression = node.expression;
-    if (expression is BinaryOperationExpression &&
-        expression.operator == BinaryOperator.dividedBy) {
+    if (node.expression
+        case BinaryOperationExpression(operator: BinaryOperator.dividedBy)) {
```

Comment:
I'd fold these together into a single `if`/`case`

Reply:
I don't think a single if/case is possible here, since we want to return in both the case where `node` is a parenthesized division expression and the case where it's not parenthesized. Does it make sense to leave this as-is or convert to a switch statement?

Reply:
Oh, I see what you mean. Yeah in this case I'd use a `switch` I think, since you're effectively enumerating cases.